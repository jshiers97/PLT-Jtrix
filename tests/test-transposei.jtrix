void main() {
	Matrix<int> a;
	int[] b;
	int i;
	int j;
	a = [[1,2,3];[4,5,6]];

	a = a.transpose_i();

	b = a.dim_i();


	for(i=0; i<b[0]; i=i++) {
		for(j=0; j<b[1]; j=j++) {
			print(a[i][j]);
		}
	}

	println("r x c");
	print(b[0]);
	print(b[1]);

	free(b);
	free(a);
}
			
