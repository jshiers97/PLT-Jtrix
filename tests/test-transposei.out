1
4
2
5
3
6
r x c
3
2
